compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -37,6 +37,11 @@ func (n *NodeInventoryCollector) Scan(nodeName string) (*storage.NodeInventory,
 		ScanTime:   timestamp.TimestampNow(),
 		Components: protoComponents,
 	}
+	// uncertifiedRHEL is false since scanning is only supported on RHCOS for now,
```

Comment:
Nit: As we add this very note to every scan for now, we could also move it into the message init, as opposed to adding it afterwards, right?
```go
	m := &storage.NodeInventory{
		NodeName:   nodeName,
		ScanTime:   timestamp.TimestampNow(),
		Components: protoComponents,
                Notes:      []scannerV1.Note{scannerV1.Note_LANGUAGE_CVES_UNAVAILABLE},
	}
```

Reply:
I second what Matthias wrote - I would also see it inside the `storage.NodeInventory` initialization block.